docs/user-guide/cai.rst

```diff
@@ -14,7 +16,7 @@ CAI allows training a harmless, but non-evasive AI assistant that engages with h
 .. _Constitutional AI (CAI): https://arxiv.org/abs/2212.08073
 
 CAI
-###############
+###
 The basic steps of CAI are described in this section and illustrated in the figure below (`Figure 1 <https://arxiv.org/abs/2212.08073>`_).
 
```

Comment:
Line 22/24

fix punctuation (fine-tuned)


(RL Stage) AI Comparison Evaluations → Reward Model → Reinforcement Learning: The AI generates pairs of responses to harmfulness prompts using the fine-tuned model, then evaluates which response is better according to a principle in the constitution, and then trains a reward model based on this dataset of AI preferences and a human helpfulness preferences. The AI then trains with RL using the learned reward model.

Reply:
https://github.com/NVIDIA/NeMo-Aligner/pull/411/commits/9432333a44ad01d9534ed89263a37000b2aaca11